### PR TITLE
7023-pwaPromptAnimation

### PR DESCRIPTION
**Pull request by lzach83**
## Link to Issue
Closes: #7023 

## Description of Changes
- Using the `useAnimation` hook, added the animation to both `IOSPrompt` and `AndroidPrompt`

## Test Plan
- Run the website locally
- Ensure the animation occurs for both prompts

### CODE REVIEW

**Review by Israellund — APPROVED** [review]
LGTM, just have to fix the one code quality test

**Comment by masvelio** [comment]
There is a moment when the modal backdrop is not fully aligned to the top bar which looks bad. Also, isn't animation too slow? Spec says 0.5s 

image


https://github.com/hicommonwealth/commonwealth/assets/14819225/90496c43-1bbe-4c1a-a3c7-d702485d54d2

File: packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/IOSPrompt/IOSPrompt.tsx [hunk 1]
```diff
@@ -15,8 +16,9 @@ export const IOSPrompt = ({
   showPrompt,
   setShowPrompt,
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/IOSPrompt/IOSPrompt.tsx:17]: remove all those warnings by removing unused props

**Review by mzparacha — APPROVED** [review]
Tested locally, LGTM. Please fix CI issues before merge.

**Review by jnaviask — CHANGES_REQUESTED** [review]
Getting the same "looks bad" as Marcin is, where the backdrop has space on the top during the animiation.

**Review by burtonator — APPROVED** [review]

**Pull request closed without merging**